src/studio/nuxeo-studio/working-in-studio/content-model/documents.md

```diff
@@ -212,10 +212,16 @@ if you don't find the default type you are looking for, you can add it [using th
 
 *   **Extends**: See the [Creating a Document Type](#creating-a-document-type) section of this page.
 *   **Label**: See the [Creating a Document Type](#creating-a-document-type) section of this page.
-*   **Category**: The category defines in which column the document type appears on the "new Document" pop up in Nuxeo DM. If you don't want to use the default categories, you can add a new one [using the custom document category registry]({{page page='registries'}}).
+*   **Category**: The category defines in which column the document type appears on the "new Document" pop up in Nuxeo Server, when using JSF UI. If you don't want to use the default categories, you can add a new one [using the custom document category registry]({{page page='registries'}}).
     ![]({{file name='available-documents-workspace.png' space='userdoc' page='creating-content'}} ?w=500,h=216,border=true)
-*   **Icon**: The icon that will be displayed in the default list view. Note that you can upload new icons on the [resources]({{page page='resources'}}) section.
-*   **Large Icon**: The icon that will be displayed in the "Available document types" pop-up and the [Icon view]({{page space='userdoc58' page='changing-workspace-content-presentation'}}) (view available up to Nuxeo Platform 5.8). Note that you can upload new icons on the& [resources]({{page page='resources'}}) section.
+*   **Icon**: The icon that will be displayed in the default list view.
+*   **Large Icon**: The icon that will be displayed in the "Available document types" pop-up when using JSF UI and the Icon view (view available up to Nuxeo Platform 5.8).
```

Comment:
The icon displayed in the "Available document types" pop-up when using JSF UI and the Icon view (view available up to Nuxeo Platform 5.8).